### PR TITLE
UI fixes 

### PR DESCRIPTION
**Pull request by srivig**
UI Fixes:
-improve filter/answer tables on metric and company pages #152889128
-consistent styling on browse pages #152899617
-fix 'Research answer' button style #152885867
-metric browse item fix
-research group contribution styles

### CODE REVIEW

File: db/migrate_cards/data/browse_source_style.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "name": "Source+description",
+    "type": "Basic"
+  },
+  {
+    "name": "source+*self+*structure",
+    "type": "HTML"
```
Comment (xithan) [inline db/migrate_cards/data/browse_source_style.json]: a Basic card is ok as a migration but a HTML card should be moved to code

File: db/migrate_cards/data/cards/source+description [hunk 2]
```diff
@@ -0,0 +1 @@
+<p><strong>All metric values on WikiRate must be traced back to a source.</strong></p><p>A source is generally a company report, including CSR Report, Sustainability Report, Annual Report or Integrated Report. It could also be a news article, website, conflict mineral report or modern slavery report and so on. A source can be added to WikiRate as a URL, file upload or report. Once a source is added it will remain of the platform so researchers can easily access the document.</p><p>{{source|add_button}}</p>
```
Comment (ethn) [inline db/migrate_cards/data/cards/source+description]: We're trying to get nests OUT of the descriptions.   We just need the HTML card to have the nest in it.

File: db/migrate_cards/data/browse_source_style.json (hunk 1 above)
Comment (srivig) [reply db/migrate_cards/data/browse_source_style.json]: There are MANY other HTML cards, I want to do it as a separate task to move them to codebase at once. This browse source related change is minor, it is similar to other browse pages. I can remove these files from this PR if they shouldn't be part of it

File: mod/metrics/template/abstract/metric_answer/content_formgroup.haml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-.panel-default.nodblclick.col-md-4#metric-container
+.panel-default.nodblclick.col-md-4.pull-left#metric-container
```
[the remark this answers is not in the document]
Comment (srivig) [reply mod/metrics/template/abstract/metric_answer/content_formgroup.haml]: I agree, I am fixing this

File: db/migrate_cards/data/cards/source+description (hunk 2 above)
Comment (srivig) [reply db/migrate_cards/data/cards/source+description]: why?

Comment (ethn) [reply db/migrate_cards/data/cards/source+description]: *Normal WikiRate users shouldn't see nests.*  

Worth saying again...

**Normal WikiRate users shouldn't see nests.**

Nests are for Deckers (Wagneers).  In most cases they should be confined to rules and structure cards.  They will seem weird and mysterious to normal users, and we don't want decko to feel mysterious.

Normal users will be free to edit descriptions and such, and many who don't want to edit may refer to these cards to see how certain things are done and who has done them.

File: db/migrate_cards/data/browse_source_style.json (hunk 1 above)
Comment (xithan) [reply db/migrate_cards/data/browse_source_style.json]: Yes, I know and working sick you have good excuse anyway but my point is when you touch a card that is wrong and should be in code according to our guidelines then you could save the time to create a migration for it and instead spend that time to get it right.

Comment (srivig) [reply db/migrate_cards/data/browse_source_style.json]: You are right, I will follow that

**Pull request merged**